app/src/main/java/com/evelynhasama/bucketbuds/FriendsPageFragment.java

```diff
@@ -136,9 +138,9 @@ public void done(List<ParseUser> objects, ParseException e) {
                 for (int i = 0; i < objects.size(); i++) {
                     friendIds.add(objects.get(i).getObjectId());
                 }
-                ParseQuery<ParseUser> query = ParseQuery.getQuery(User.USER_CLASS);
-                query.whereNotEqualTo(User.KEY_OBJECT_ID, User.getCurrentUser().getObjectId());
-                query.whereStartsWith(User.KEY_USERNAME, queryPrefix);
```

Comment:
Why not use the `User` constants if you are querying `Users`?

Reply:
good point!